rustls/src/client/handy.rs

```diff
@@ -251,6 +252,7 @@ test_for_each_provider! {
     use crate::msgs::handshake::SessionId;
     use crate::msgs::persist::Tls13ClientSessionValue;
     use crate::suites::SupportedCipherSuite;
+    use crate::msgs::base::PayloadU16;
```

Comment:
Worth a mention in the commit message to help motivate the choice of `PayloadU16` vs `Vec`?